lib/components/AppMainView.jsx

```diff
@@ -65,16 +65,16 @@ const AppMainView = ({
         <WarningView />
         <div className="memory-box-container">
             <MemoryBoxView
-                title={getTargetTitle(serialNumber, deviceInfo) || 'Device Memory Layout'}
+                title={getTargetTitle(serialNumber, deviceInfo) || 'Device memory layout'}
```

Comment:
Looking through our source we do not have a coherent guideline whether or when to use title case, right?

Reply:
I just noticed: Where I apparently discovered title case, it was in fact only app names that occurred in other strings. So it seems to me now that we only use title case for app names, but otherwise only sentence case in UI elements.

Reply:
I don't think we have any written rules, but so far BLE app should be the guiding example until rules gets written.